plugins/state_history_plugin/CMakeLists.txt

```diff
@@ -6,4 +6,6 @@ add_library( state_history_plugin
 target_link_libraries( state_history_plugin state_history chain_plugin eosio_chain appbase )
 target_include_directories( state_history_plugin PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" )
 
-add_subdirectory(tests)
+if( CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 11 )
+   target_compile_options( state_history_plugin PRIVATE "-fcoroutines" )
+endif()
```

Comment:
The situation with gcc 10 and corountines is certainly a bit... eyebrow raising. So far I've not seen any indications it is malfunctioning in our usage.